README.md

```diff
@@ -190,25 +194,54 @@ It provides a trivially simple REST API:
 
 A capped collection can be used in MongoDB to limit the storage. For example to store only the last 1000 replies received.
 
-### Scale and message ordering - nonce management
+### Nonce management for Scale and Message Ordering
 
-The transaction pooling/execution logic within an Ethereum node is based upon the concept of a `nonce`, which must be incremented exactly once each time a transaction is submitted from the same Ethereum address. There can be no gaps in the nonce values, or messages build up in the `queued transaction` pool waiting for the gap to be filled. This allows for deterministic ordering of transactions sent by the same sender.
+The transaction pooling/execution logic within an Ethereum node is based upon the concept of a `nonce`, which must be incremented exactly once each time a transaction is submitted from the same Ethereum address. There can be no gaps in the nonce values, or messages build up in the `queued transaction` pool waiting for the gap to be filled (which is the responsibility of the
+sender to fill). This allows for deterministic ordering of transactions sent by the same sender.
 
 The management of this `nonce` pushes complexity back to the application tier - especially for horizontally scaled Enterprise applications sending many transactions using the same sender. By using an ordered Messaging stream to submit messages into the Ethereum network, many applications are able to delegate this complexity to kaleido-io/ethconnect.
 
 The kaleido-io/ethconnect bridge contains all the logic necessary to communicate with the node to determine the next nonce, and also to cope with multiple requests being in flight within the same block, for the same sender (including [with IBFT](https://github.com/ethereum/EIPs/issues/650#issuecomment-360085474)).
 
+When using the bridge, an application can submit simple YAML/JSON formatted transactions
```

Comment:
I'm assuming here that we are doing some very aggressive nonce checking? Checking the txpool before submitting new transactions and blocks?
What if instead of filling the nonce gap with new transactions my intention is to increase the gas expenditure on transactions stuck in the txpool so that miners will process them?

Reply:
At the moment the code is optimised for the Enterprise permissioned chain scenario, where there is no Ether economy for mining. We do not replace transactions that have been accepted into the pending transactions pool. When there's no mining economy these should not get stuck due to a miner deciding the gasPrice is too low (the gasPrice should be zero). However, if there is an unfunded account a transaction could get stuck there. There's no special code here to manage a transaction that blocks up a whole sender because the sender needs funding.